Activities/ManagePersonsPermissionCheckerTask.cs

```diff
@@ -1,74 +1,75 @@
+using Lombiq.TrainingDemo.Permissions;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.Extensions.Localization;
-using OrchardCore.Security.Services;
 using OrchardCore.Users.Models;
 using OrchardCore.Users.Services;
 using OrchardCore.Workflows.Abstractions.Models;
 using OrchardCore.Workflows.Activities;
 using OrchardCore.Workflows.Models;
 using OrchardCore.Workflows.Services;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
 
 namespace Lombiq.TrainingDemo.Activities;
 
-// A simple workflow activity task that accepts an username as a TextField input and checks whether the user has
-// ManagePersons Permission or not.
+// A simple workflow task that accepts a username as a TextField input and checks whether the user has ManagePersons Permission or not.
```

Comment:
This training section isn't hooked into the walkthrough. Link it from the end of the previous one (GraphQL).